src/harness/testcases/grant_testcase.py

```diff
@@ -88,6 +88,127 @@ def test_WINNF_FT_S_GRA_1(self):
       self.assertEqual(response['channelType'], 'GAA')
       self.assertEqual(response['response']['responseCode'], 0)
 
+  @winnforum_testcase 
+  def test_WINFF_FT_S_GRA_4(self):
+    """Successful CBSD grant request.
+        No incumbent present in the PAL frequency range requested by the CBSD.
+    """
+    # load the device
+    device_a = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    self._sas_admin.InjectFccId({'fccId': device_a['fccId']})
+    # load PAL Database Record
+    pal_record = json.load(
+        open(os.path.join('testcases', 'testdata', 'pal_record_0.json')))
+    # load PPA record
+    ppa_record = json.load(
+        open(os.path.join('testcases', 'testdata', 'ppa_record_0.json')))
+
+    #change device location to be inside PPA
+    device_a['installationParam']['latitude'], \
+    device_a['installationParam']['longitude'] = getRandomLatLongInPolygon(ppa_record)
+    #register device
+    request = {'registrationRequest': [device_a]}
+    response = self._sas.Registration(request)['registrationResponse'][0]
+    # Check registration response
+    self.assertEqual(response['response']['responseCode'], 0)
+    cbsd_id = response['cbsdId']
+    del request, response
+
+    pal_low_frequency = 3620000000.0
+    pal_high_frequency = 3630000000.0
+
+    #Make PPA and PAL records
+    ppa_record, pal_record = makePpaAndPalRecordsConsistent(ppa_record,[pal_record],\
+        pal_low_frequency,pal_high_frequency,device_a['userId'])
+
+    #Add cbsd reference to PPA
+    ppa_record['ppaInfo']['cbsdReferenceId'] = [cbsd_id]
+    
+    # Inject PAL record record
+    self._sas_admin.InjectPalDatabaseRecord(pal_record[0])
+
+    # Inject PPA record with cbds_id reference
+    ppa_record = {'zoneData':ppa_record}
```

Comment:
Can you change the key from zoneData to record
https://docs.google.com/document/d/19YbeQsSzm5kCIOeVzsF8piqI0Lizy3p345aiVGf-Hlg/edit#heading=h.fbgibud28jgj